### PR TITLE
x11-base/xorg-server: revbump to 1.20.5

### PR DESCRIPTION
**Pull request by stefson**
`xorg-server-1.20.5` got stable recently

### CODE REVIEW

**Comment by stefson** [comment]
where is it? did you forgot to push?

**Comment by blueness** [comment]
4e7fd6f

**Comment by blueness** [comment]
Somethng went  wrong.  If you look above, you can see that it was merged, but it never landed.  Could this be a github bug?

Let me try again.

**Comment by blueness** [comment]
its there now.  do you have any idea what might have happened?

**Comment by stefson** [comment]
I have no idea. But it seems the same happened with #256

**Comment by blueness** [comment]
Did you fork from the gentoo/musl overlay to make the PR?

Anyhow #256 is pushed.  Please look for any other PRs that have been missed.
